examples/init/abacus/fcc-Al-lcao/machine.json

```diff
@@ -0,0 +1,106 @@
+{
+  "api_version": "1.0",
+  "deepmd_version": "2.0.1",
+  "train" :[
+    {
+      "command": "dp",
+      "machine": {
+        "batch_type": "",
```

Comment:
Why is it empty?

Reply:
I thought it depends on users' setting, so left it empty. Is it better to add my setting, "Lebesgue"?